statistics/histogram.go

```diff
@@ -139,8 +139,10 @@ func (hg *Histogram) MemoryUsage() (sum int64) {
 	if hg == nil {
 		return
 	}
-	sum = EmptyHistogramSize + hg.Bounds.MemoryUsage() + int64(cap(hg.Buckets))*EmptyBucketSize + int64(cap(hg.scalars))*EmptyScalarSize
-	return
+	if len(hg.Buckets) == 0 && len(hg.scalars) == 0 && hg.Bounds.Capacity() == 0 {
+		return
+	}
+	return hg.Bounds.MemoryUsage()
```

Comment:
What's the purpose of this change?

Reply:
We found the original calculation has large diff from the test in https://pingcap.feishu.cn/docx/doxcnZ0e75oMBm959tPu7NL8Dag#doxcn4QOMAuk6icayQX6KmGmY3f